#!/bin/sh /usr/share/dpatch/dpatch-run
## 17-magic-remove-quicktime.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: Disabled some QuickTime entries (ASCII words, closes: #366986, #308394).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/animation file-4.20/magic/Magdir/animation
--- file-4.20.orig/magic/Magdir/animation	2007-01-14 18:47:21.000000000 +0000
+++ file-4.20/magic/Magdir/animation	2007-03-27 12:23:14.000000000 +0000
@@ -14,11 +14,11 @@
 >12     string          cmov            \b movie (fast start, compressed header)
 >12     string          rmra            \b multiple URLs
 4       string          mdat            Apple QuickTime movie (unoptimized)
-4       string          wide            Apple QuickTime movie (unoptimized)
-4       string          skip            Apple QuickTime movie (modified)
-4       string          free            Apple QuickTime movie (modified)
+#4       string          wide            Apple QuickTime movie (unoptimized)
+#4       string          skip            Apple QuickTime movie (modified)
+#4       string          free            Apple QuickTime movie (modified)
 4       string          idsc            Apple QuickTime image (fast start)
-4       string          idat            Apple QuickTime image (unoptimized)
+#4       string          idat            Apple QuickTime image (unoptimized)
 4       string          pckg            Apple QuickTime compressed archive
 4	string/B	jP		JPEG 2000 image
 4	string		ftyp		ISO Media
